config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
   # root "articles#index"
+  #root 'users#index'
+  # Defines the root path route ("/")
+  resources :users, :posts, only: [:index, :show]
```

Comment:
- [ ] We need to define these routes more efficiently. The posts routes should be nested within the users routes so we can access the post's author id. Kindly refactor to something like this:

  ```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
  ```